.gitignore

```diff
@@ -40,3 +40,6 @@ yarn-error.log*
 # Integrated development environnements
 .idea
 *.sublime-*
+
+# Compiled locales
+#src/locales/*/*.js
```

Comment:
why do we not want to use compiled locales?

Reply:
not sure how others see this, but the `t` method has been pretty standard amongst most projects i have seen that have implement internationalization...is clunky though, agree :-)